packages/dbos-cloud/applications/register-app.ts

```diff
@@ -11,14 +11,16 @@ export async function registerApp(dbname: string, host: string, appName?: string
   if (!appName) {
     return 1;
   }
-  logger.info(`Registering application: ${appName}`);
+  const appLanguage = retrieveApplicationLanguage();
```

Comment:
If the language is python, shouldn't we mandate a `requirements.txt` during deploy?